tunnel/tunnel.go

```diff
@@ -124,27 +50,29 @@ func StartReverseTunnel(tunnelConfig *Config, wg *sync.WaitGroup) {
 	)
 
 	go func() {
-		<-c
-		cleanup(tunnelConfig)
-		os.Exit(0)
+		<-s
+		if semaphore.CanRun() {
+			cleanup(tunnelConfig)
+			os.Exit(0)
+		}
 	}()
 
-	fmt.Printf("\rNow forwarding localhost:%s to %s://%s.%s\n",
-		tunnelConfig.LocalPort, tunnelConfig.EndpointType, tunnelConfig.Subdomain, tunnelConfig.EndpointURL.Host)
+	fmt.Printf("Access your website at %s://%s.%s\n",
```

Comment:
I'm not sure about this about. We could be forward any type of server, right? I think the most common use case will probably be websites, but anyone savvy enough to be using the tool probably realizes that they're forwarding a port. There's probably a good compromise between the old output and the new.

Reply:
We only forward websites right now